src/lib/core/ripple/ripple.ts

```diff
@@ -102,11 +107,11 @@ export class MdRipple implements OnChanges, OnDestroy {
 
   /** Ripple configuration from the directive's input values. */
   get rippleConfig(): RippleConfig {
-    return {
+    return extendObject({
       centered: this.centered,
       speedFactor: this.speedFactor,
       radius: this.radius,
       color: this.color
-    };
+    }, this._globalRippleConfig);
```

Comment:
Ok then maybe this PR is conflating two things: the per-ripple options, and the global ripple options. I agree that disabled doesn't make sense for individual ripples, and I think speed factor doesn't make sense for the global settings. The global setting should be the base speed, and then individual ripples define a speed factor that is multiplied together with the global base speed. That way we preserve the relative speeds when setting the global.